specification/quota/resource-manager/Microsoft.Quota/stable/2025-09-01/quota.json

```diff
@@ -0,0 +1,1299 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Usage properties for the specified resource.",
+          "$ref": "#/definitions/UsagesProperties"
+        }
+      }
+    },
+    "UsagesProperties": {
+      "description": "Usage properties for the specified resource.",
+      "type": "object",
+      "properties": {
+        "usages": {
+          "description": "The quota limit properties for this resource.",
+          "$ref": "#/definitions/UsagesObject"
+        },
+        "unit": {
+          "description": " The units for the quota usage, such as Count and Bytes. When requesting quota, use the **unit** value returned in the GET response in the request body of your PUT operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name provided by the resource provider. Use this property name when requesting quota.",
+          "$ref": "#/definitions/ResourceName"
+        },
+        "resourceType": {
+          "description": "The name of the resource type. Optional field.",
+          "$ref": "#/definitions/ResourceTypesName"
+        },
+        "quotaPeriod": {
+          "description": "The time period for the summary of the quota usage values. For example:\r\n*P1D (per one day)\n*PT1M (per one minute)\n*PT1S (per one second).\r\nThis parameter is optional because it is not relevant for all resources such as compute.",
+          "type": "string",
+          "readOnly": true
+        },
+        "isQuotaApplicable": {
+          "description": "States if quota can be requested for this resource.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Additional properties for the specific resource provider.",
+          "type": "object"
+        }
+      }
+    },
+    "UsagesLimits": {
+      "type": "object",
+      "description": "Quota limits.",
+      "properties": {
+        "value": {
+          "description": "List of quota limits.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CurrentUsagesBase"
+          }
+        },
+        "nextLink": {
+          "description": "The URI used to fetch the next page of quota limits. When there are no more pages, this is null.",
+          "type": "string"
+        }
+      }
+    },
+    "CurrentQuotaLimitBase": {
+      "type": "object",
+      "description": "Quota limit.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The resource name.",
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
can we inherit "allOf" ProxyResource instead of redeclaring all the resource properties?